src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
This property is a "documented" one - http://api.jquery.com/on/, also there might be some use-cases when `originalEvent` could be useful. 

I'd say commit that started this whole thing is just to risky.